### DISCUSSION TITLE
Some examples are broken on bulletphysics.org

### DISCUSSION THREAD

**Discussion by szabolcsdombi** [bulletphysics/bullet3#3541] [General]
On the [Collision_Callbacks_and_Triggers](http://www.bulletphysics.org/mediawiki-1.5.8/index.php?title=Collision_Callbacks_and_Triggers) page the following error occurres:

`Process 'highlight --fragment --syntax='cpp' --style-outfile='/tmp/highlight.css'' failed to start?`

Example code is not readable.

**Comment by benelot** [comment]
True, something broke the wiki.

**Comment by szabolcsdombi** [comment]
seems to work fine, thank you!

**Comment by erwincoumans** [comment]
At some stage I'd like to retire the bulletphysics.org/wiki and move to use github.com/bulletphysics/bullet wiki. That way, I (or someone else) can review all content, make sure it doesn't conflict with existing documentation etc.

**Comment by benelot** [comment]
Sounds like a good idea! I will start to port my articles soon and create
new ones there.

**Comment by a-day-old-bagel** [comment]
So, was this going to get fixed? It's still broken on every page on the wiki for me, in both chrome and firefox. It makes it very hard to learn anything about bullet, since some of the topics don't appear to be in the manual (or am I wrong?)

**Comment by benelot** [comment]
No, these things are not in the manual. I have no idea what might have
caused this. Did you do any changes on the server recently? Because it
seems that cpp as a language is gone on one hand and that changing the
language to something like c does not work either. The highlight command
seems to be broken.

**Comment by Banderi** [comment]
This really is a bummer. Is there no really nothing that could fix the command, or could it be temporarily disabled to see the raw text? I'm looking forward to the articles being ported though, can't wait for it! If I were any competent in which are correct and which are outdated I'd offer my full support and help in doing so.

**Comment by benelot** [comment]
I can understand that it is very annoying, as it totally inhibits you from reading the code sections...

@Banderi: I can give you a list of those that make sense to be ported. We could then remove them one by one from the old wiki and just leave a link to the new one on the old pages. This would stop people from editing the old ones.

**Comment by benelot** [comment]
It seems to work again.
